docs/design/FreeCreditOfferTypesDesignDoc.md

```diff
@@ -0,0 +1,229 @@
+
+**Referral Offer Credit**: An incentive program for motivating existing clients to refer their peers to set up paid accounts on the Tardigrade network. 
+Existing clients give their UUID Referral Link to their friends for their friends to create an account and receive an Invite Offer Credit that is greater than the default Free Tier Credit.
+
+## Background
+
+### REFERRAL LINK
+
+- When a user wants to issue a referral link:
+  An existing client who has an account on one of the Tardigrade Satellites is able to retrieve their unique referral link from their dashboard. They can issue their referral link to their friends in the following ways:
+  - Copy the static link associated to their account with the default invite message from their dashboard 
+  Twitter Share
+
+- When a referral link is accepted:
+  The invitee can accept their invite to redeem their Referral Offer by going through the Referral Link to create an account on the Satellite they were invited to/through.
+  Upon creating their account they must enter their payment information into the Stripe integration and accept/acknowledge that:
+    If they use up their referral credit before the expiration date they should expect a bill for any amount of activity they acquire beyond what was credited to them
+    When their offer period expires they should expect a bill for any activity on the Tardigrade network going forward billed monthly at unit per hour
+
+### AWARDED CREDIT
+
+- When free credit is awarded:
+It is displayed as the USD value that is equivalent to the storage & bandwidth allotted amount for the credit
+The credit is automatically applied to the account and will have a max limit that can be reached within a set timeframe, as well as an expiration date for it to be used by.
+  The user will be charged for activity that is beyond the Free Credit limit within the set timeframe
+  The user will be charged for activity that is after the expiration date.
+  Example statement to the user: 
+  “You will not be charged until all of your $XX credit is used or it expires. Your $XX credit will be applied immediately”
+  A user's free credit can only be applied to projects that they create
+
+  A client can see their awarded Free Credits in their dashboard with associated expiration date
+  A client who has been awarded storage credit for their referrals is able to retain the awarded credits when the referral offer they received the credit under has been updated/changed
+
+### CATEGORIES OF FREE CREDIT & OFFER TYPES
+
+**Default Free Tier Offer:**
+  - This is the credit that all new users who join the network receive if they create an account without a Referral Link. Once a new user successfully creates their account they will have 14 days to use their Free Credit allowance.  
+  The Default Offer will remain constant, with no expiration date to the offer and does not have a redemption cap
+  The Offer Program Admin is able to adjust the default offer
```

Comment:
Adjust in terms of amount? Is Default Offer the same as Referral Offer Credit?

Reply:
Yes, they will adjust the amount, when it should expire. The credits are all the same but just have different expiration date